kyverno-policy-reporter-ui.yaml

```diff
@@ -34,25 +38,28 @@ pipeline:
   - working-directory: frontend
     runs: |
       bun install
-      bun run build
-
-      mkdir -p ${{targets.destdir}}/app
-      cp -R ./dist ${{targets.destdir}}/app/dist
+      bun run generate
+      mkdir -p ${{targets.contextdir}}/var/run/ko/templates
+      cp -R ./dist ${{targets.contextdir}}/var/run/ko/ui
 
   - uses: go/build
     with:
       modroot: backend
       packages: .
       output: policyreporter-ui
 
+  - name: copy more frontend requirements
+    runs: |
+      cp -R backend/templates ${{targets.contextdir}}/var/run/ko/
+
 subpackages:
   - name: kyverno-policy-reporter-ui-compat
     description: compatibility symlinks package for policyreporter-ui Dockerfile
     pipeline:
       - runs: |
-          # Symlink the binary from usr/bin to /app/
-          mkdir -p ${{targets.subpkgdir}}/app
-          ln -sf /usr/bin/policyreporter-ui ${{targets.subpkgdir}}/app/policyreporter-ui
+          # Symlink the binary from usr/bin to /ko-app/
+          mkdir -p ${{targets.contextdir}}/ko-app
+          ln -sf /usr/bin/policyreporter-ui ${{targets.contextdir}}/ko-app/policyreporter-ui
```

Comment:
Can we have a `readlink` test here?
Is this a breaking change?

Maybe we should keep the old link, just in case?

Reply:
thank you, I think this was probably an oversight in the first go itself with `/ko-app` being `/app` but keeping it here to make it more backwards compatible. 
this is a breaking change per se but it's not working otherwise so I'll say it's good that we're now more closely inline with the project.